### ISSUE TITLE
Wrong date format delivered by twisted.conch.ls.lsLine

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3503]
|<img alt="Bombenbodo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Bombenbodo reported|
|-|-|
|Trac ID|trac#3503|
|Type|defect|
|Created|2008-10-26 19:31:55Z|
|Branch|https://github.com/twisted/twisted/tree/lsline-3503|

A simple test script running at "Python 2.5.2 (r252:60911, Feb 21 2008, 13:11:45) [MSC v.1310 32 bit (Intel)] on win32"

```
import time
from twisted.conch.ls import lsLine

class st:
    pass

s = st()
name = 'test'
s.st_mode = 040755
s.st_uid = 500
s.st_gid = 1000
s.st_size = 0
s.st_atime = time.time()
s.st_mtime = time.time()
s.st_nlink = 1
print lsLine(name, s)
```

results in 

```
drwxr-xr-x    1 500      1000            0 Oct d 20:30 test
```

but it should be

```
drwxr-xr-x    1 500      1000            0 Oct 26 20:30 test
```

Proposed solution: change '%2d' in '%d' in lines 50 and 52 of twisted.conch.ls.lsLine
```
...
if s.st_mtime + sixmo < time.time(): # last edited more than 6mo ago
        # bugfix
        # l += time.strftime("%b %2d  %Y ", time.localtime(s.st_mtime))
        l += time.strftime("%b %d  %Y ", time.localtime(s.st_mtime))
    else:
        # bugfix
        # l += time.strftime("%b %2d %H:%S ", time.localtime(s.st_mtime))
        l += time.strftime("%b %d %H:%S ", time.localtime(s.st_mtime))
    l += name
    return l
```

<details><summary>Searchable metadata</summary>

```
trac-id__3503 3503
type__defect defect
reporter__Bombenbodo Bombenbodo
priority__normal normal
milestone__ 
branch__branches_lsline_3503 branches/lsline-3503
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1225049515000000 1225049515000000
changetime__1227015847000000 1227015847000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [25399]) Branching to 'lsline-3503'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Okay, should be fixed.  Build results http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/lsline-3503

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

Cool, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [25412]) Merge lsline-3503

Author: exarkun
Reviewer: therve
Fixes: [#3503](https://github.com/twisted/twisted/issues/3503)

Fix the status listing formatting function `twisted.conch.ls.lsLine` so that
it formats days properly on Windows and so that it includes the actually value
for the minute field in the minute field instead of using the value for the
seconds field instead.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
